checker-qual/src/main/java/org/checkerframework/checker/sqlquotes/qual/SqlEvenQuotes.java

```diff
@@ -0,0 +1,22 @@
+package org.checkerframework.checker.sqlquotes.qual;
+
+import java.lang.annotation.*;
+import org.checkerframework.framework.qual.QualifierForLiterals;
+import org.checkerframework.framework.qual.SubtypeOf;
+
+/**
+ * SqlEvenQuotes: Used to denote a String that comprises part of a SQL query and contains either
+ * zero or an even number of unescaped single quotes – i.e., there must be either zero or an even
+ * number of ‘ characters in a SqlEvenQuotes String that are not preceded immediately by a \
```

Comment:
The discussion of quoting is a bit confusing because it's easy for a reader to think that the `\` you are talking about is Java quoting.  But it's not -- it is SQL quoting.  Please make this a bit more explicit.